accurics-aws-tf-sample.tf

```diff
@@ -171,10 +184,11 @@ resource "aws_kms_key" "accurics-test-kmskey1" {
   description             = "accurics-test-kmskey1"
   deletion_window_in_days = 30
   tags = {
-    Name = format("%s-kmskey1", var.acqaPrefix)
+    Name         = format("%s-kmskey1", var.acqaPrefix)
     ACQAResource = "true"
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: App-Tier CloudWatch Log Group Retention Period**
 
**Vulnerability**
App-Tier CloudWatch Log Group Retention Period
 
**Remediation**
Ensure CloudWatch log group for app tier has a retention period.

source: accurics-app